### PR TITLE
Ouerea flags

### PR DESCRIPTION
**Pull request by Generalcamo**
Adds two flags to represent Ouerea, one old (but official) and one current (but not recognized by the Hegemony).

Based on the flags by Caelphon, with one edited by Triogenix.

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Comment by Triogenix** [comment]
This has my approval as Unathi Lore Maintainer.

**Review by Arrow768 — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**
